src/target_enums.cpp

```diff
@@ -332,6 +332,26 @@ std::string OSToString(TargetOS os) {
     UNREACHABLE();
 }
 
+std::string OSToLowerString(TargetOS os) {
+    switch (os) {
+    case TargetOS::windows:
+        return "window";
```

Comment:
Please fix "window"->"windows"

Reply:
That's really good catch. Thanks! Fixed.